debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/ViewSelectedColumnsParserListener.java

```diff
@@ -108,15 +109,15 @@ private TableEditor parseSelectElements(MySqlParser.SelectElementsContext ctx) {
         TableEditor table = Table.editor();
         if (ctx.star != null) {
             tableByAlias.keySet().forEach(tableId -> {
-                table.addColumns(tableByAlias.get(tableId).columns());
+                table.addColumns(tableByAlias.get(tableId).columns().collect(Collectors.toList()));
```

Comment:
The `Table#columns()` method now returns a `Stream` rather than the `List<Column>` and we use this stream to collect them into a new `ArrayList<>` in a variety of places in this PR.  I'm curious how much GC overhead that might lead to with repeatedly creating a new collection?

Reply:
Yes, great question! I _think_ the toList() call only happens off the hot path, i.e. not DML events, but only DDL events and testing. Worth double checking, but if so indeed, I think this is acceptable. In general I think, the actual change event handling loop is where we need to be careful, for DDL events things are less critical.

Reply:
So I double checked, the `toList()` conversion doesn't happen for DML events, so this should be fine.